src/goto-programs/json_goto_trace.cpp

```diff
@@ -118,6 +118,10 @@ void convert_decl(
   const symbolt *symbol;
   irep_idt base_name, display_name;
 
+  DATA_INVARIANT(
```

Comment:
Compare with the corresponding code in goto_trace.cpp: https://github.com/diffblue/cbmc/blob/557be1d344d01cb2826deaff12e3a65222f93b03/src/goto-programs/goto_trace.cpp#L271